packages/manager/modules/pci/src/components/project/images-list/images-list.service.js

```diff
@@ -3,12 +3,28 @@ import groupBy from 'lodash/groupBy';
 import map from 'lodash/map';
 import pick from 'lodash/pick';
 import omit from 'lodash/omit';
+import snakeCase from 'lodash/snakeCase';
 
 import Image from './images.class';
+import { IMAGES_REGEX } from './images.constants';
+
+function getDistribution(name, type) {
+  const os = IMAGES_REGEX[type];
+  if (os) {
+    const distribution = os.find(
+      (distrib, index) => index < os.length && distrib.regex.test(name),
```

Comment:
why do we need the check `index < os.length`? all elements of the array will definitely will have the index less than the length :)